.clusterfuzzlite/build.sh

```
@@ -0,0 +1,126 @@
+mvn dependency:copy-dependencies \
+    -DoutputDirectory=target/deps \
+    -DincludeScope=test -B -q
+
+# ── Stage all runtime artifacts into $OUT ────────────────────────
+
+# Project classes and test classes
+cp -r target/classes $OUT/classes
+cp -r target/test-classes $OUT/test-classes
+
+# All dependency JARs
+mkdir -p $OUT/deps
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)